packages/@ember/array/index.ts

```diff
@@ -1224,29 +1021,17 @@ const EmberArray = Mixin.create(Enumerable, {
       }); // false (true && false && false)
     ```
 
-    @method reduce
-    @param {Function} callback The callback to execute
-    @param {Object} initialValue Initial value for the reduce
-    @return {Object} The reduced value.
+    @method isAny
```

Comment:
Whoops, ended up with the wrong doc comment annotations copied over here—you have the `isAny` annotations on `reduce`!

Reply:
Oops! 😅 
Fixed it.